one_fm/templates/emails/default_email_with_workflow.html

```diff
@@ -44,7 +44,9 @@ <h2 style="text-align: center;"> {{header}} </h2>
                     {% endif %}
                     <br>
                     The link to the document is  <a target="_blank" class="underline" href="{{ doc_link }}"> {{reference_name}}</a> <br>
-                    Please take your action for document using the action button below.<br>
+                    {% if attachments is not none%}
+                        Please take your action for document using the action button below.<br>
+                    {% endif %}
                 </p>
                 {% for action in actions %}
                     <a href="{{ action.action_link }}" class="btn btn-primary btn-action">{{ action.action_name }}</a>
```

Comment:
The Cancel button is always there can you look into it ` <a href="/" class="btn btn-secondary btn-action">Cancel</a>`

Reply:
I think it's because it exists in workflow action, no?